bootcamp-milestone-2/src/app/components/project.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import style from './blogPreview.module.css'
+import Link from 'next/link'
+import Image from 'next/image'
+import type { Project } from '@/app/typings/project'
+
+export default function ProjectPreview(props: Project) {
+  return (
+      <div className='project'>
+            <h2 className="project-title">{props.title}</h2>
+            <h3 className="project-date">{props.date}</h3>
+            <Link href={props.slug}>
+                    <Image 
+                        src={props.imgSlug}
+                        width={props.imgWidth}
+                        height={props.imgHeight}
+                        alt={props.imgDesc}>
+                    </Image>
+            </Link>
+            <p className="project-description">{props.description}</p>
+            <Link href={`/project/${props.slug}`}> Learn More </Link>
```

Comment:
make sure this link works or change/remove it. the urls are based on the names of the folders. look at how your urls for the blog pages match the folder names for an example